exercises/diffie-hellman/diffie_hellman_test.py

```diff
@@ -1,56 +1,40 @@
 import unittest
 
-import diffie_hellman
-
+from diffie_hellman import private_key, public_key, secret
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
 
-class DiffieHellmanTest(unittest.TestCase):
 
-    def test_private_key_is_in_range(self):
+class DiffieHellmanTest(unittest.TestCase):
+    def test_private_key_is_in_range_1_p(self):
         primes = [5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
         for i in primes:
-            self.assertTrue(1 < diffie_hellman.private_key(i) < i)
+            self.assertTrue(1 < private_key(i) < i)
 
-    # Can fail due to randomness, but most likely will not,
-    # due to pseudo-randomness and the large number chosen
```

Comment:
This comment may be included as a function docstring inside the `elif property == "private_key_is_random"` block.